functions/tips/main.py

```diff
@@ -50,13 +52,52 @@ def function_specific_computation():
     return 1
 
 
+def light_computation():
+    numbers = list(range(1, 10))
+    return reduce(lambda x, t: t + x, numbers)
+
+
+def heavy_computation():
+    # Multiplication is more computationally expensive than addition
+    numbers = list(range(1, 10))
+    return reduce(lambda x, t: t * x, numbers)
+
+
 # [START functions_tips_lazy_globals]
 # Always initialized (at cold-start)
 non_lazy_global = file_wide_computation()
 
 # Declared at cold-start, but only initialized if/when the function executes
 lazy_global = None
 
+# [START functions_tips_scopes]
+# [START run_tips_global_scope]
+# Global (instance-wide) scope
+# This computation runs at instance cold-start
+instance_var = heavy_computation()
+# [END functions_tips_global_scope]
```

Comment:
Can you wrap this code in one region tag not two separate chunk?

Reply:
Also it looks like you put this code in the middle of another region tag.

Reply:
Yikes. Thanks for catching that.
The latest diff should be cleaner.